go.mod

```diff
@@ -23,12 +23,12 @@ require (
 	github.com/aquasecurity/table v1.8.0
 	github.com/aquasecurity/testdocker v0.0.0-20230111101738-e741bda259da
 	github.com/aquasecurity/tml v0.6.1
-	github.com/aquasecurity/trivy-aws v0.7.0
+	github.com/aquasecurity/trivy-aws v0.7.1-0.20240117025219-411199f008cd
```

Comment:
TODO: Use the next release after https://github.com/aquasecurity/trivy-aws/pull/94 is merged.